app/Data/Profile/ProfileData.php

```diff
@@ -1047,17 +1047,7 @@ private function getLeagueData(){
         ->where('region', $this->region);
 
       // @phpstan-ignore-next-line
-      $mmr_data = \App\Models\MasterMMRDataAR::select("game_type", "conservative_rating", "win", "loss")
-        ->where('type_value', '10000')
-        ->where('game_type', '6')
-        ->where('blizz_id', $this->blizz_id)
-        ->where('region', $this->region)
-        ->union($qm)
-        ->union($ud)
-        ->union($hl)
-        ->union($tl)
-        ->union($sl)
-        ->get();
+      $mmr_data = \App\Models\MasterMMRDataAR::select("game_type", "conservative_rating", "win", "loss")->where('type_value', '10000')->where('game_type', '6')->where('blizz_id', $this->blizz_id)->where('region', $this->region)->union($qm)->union($ud)->union($hl)->union($tl)->union($sl)->get();
```

Comment:
Why does this get condensed into a single line?

Reply:
Static analysis does not match the correct types for $qm, etc, so the function calls to `union()` fail analysis. In order to ignore I had to put it on the same line. I will do some more research on multi-line ignoring but I couldn't find out a way to do it initially.

Reply:
Confirmed this is actually a bug with PHPStan. I recommend leaving the single line for now, even though it's ugly, and we can revisit the formatting when the upstream issue is resolved.

https://github.com/phpstan/phpstan/issues/4250